framework/src/dirackernels/DiracKernel.C

```diff
@@ -178,7 +190,27 @@ DiracKernel::computeQpOffDiagJacobian(unsigned int /*jvar*/)
 void
 DiracKernel::addPoint(const Elem * elem, Point p, unsigned /*id*/)
 {
-  if (!elem || (elem->processor_id() != processor_id()))
```

Comment:
I see now that we silently ignored null elements before so if we do have tests that rely on this, then I'm fine having the default be `IGNORE`

Reply:
Yeah, that was relied on by `point_caching_moving_mesh.i`.